### ISSUE TITLE
loss一开始就为nan是什么原因

### ISSUE THREAD

**Issue by zzz2923** [bubbliiiing/deeplabv3-plus-pytorch#132]
d:\graduate_student\paper\model\deeplabv3+_pytorch\deeplabv3+_stage1\train.py "
initialize network with normal type
Load weights model_data/deeplab_mobilenetv2.pth.

Successful Load Key: ['backbone.features.0.0.weight', 'backbone.features.0.1.weight', 'backbone.features.0.1.bias', 'backbone.features.0.1.running_mean', 'backbone.features.0.1.running_var', 'backbone.features.0.1.num_batches_tracked', 'backbone.features.1.conv.0.weight', 'backbone.features.1.conv.1.weight', 'backbone.features.1.conv.1.bias', 'backbone.features.1.conv.1.running_mean', 'backbone.features.1.conv.1.running_var', 'backbone.features.1.conv.1.num_batches_tracked', 'backbone.features.1.conv.3.weight', 'bac ……
Successful Load Key Num: 369

Fail To Load Key: ['cls_conv.weight', 'cls_conv.bias'] ……
Fail To Load Key num: 2

温馨提示，head部分没有载入是正常现象，Backbone部分没有载入是错误的。
Configurations:
----------------------------------------------------------------------
|                     keys |                                   values|
----------------------------------------------------------------------
|              num_classes |                                        2|
|                 backbone |                                mobilenet|
|               model_path |       model_data/deeplab_mobilenetv2.pth|
|              input_shape |                               [512, 512]|
|               Init_Epoch |                                        0|
|             Freeze_Epoch |                                       50|
|           UnFreeze_Epoch |                                      100|
|        Freeze_batch_size |                                        4|
|      Unfreeze_batch_size |                                        2|
|             Freeze_Train |                                     True|
|                  Init_lr |                                   0.0001|
|                   Min_lr |                   1.0000000000000002e-06|
|           optimizer_type |                                      sgd|
|                 momentum |                                      0.9|
|            lr_decay_type |                                      cos|
|              save_period |                                        5|
|                 save_dir |                                     logs|
|              num_workers |                                        0|
|                num_train |                                     1741|
|                  num_val |                                      194|
----------------------------------------------------------------------
Start Train
Epoch 1/100: 100%|████████████████████████████████████████████████████████████████████████| 435/435 [07:41<00:00,  1.06s/it, f_score=0.422, lr=5e-5, total_loss=nan] 
Finish Train
Start Validation
Epoch 1/100: 100%|█████████████████████████████████████████████████████████████████████████| 48/48 [00:09<00:00,  5.03it/s, f_score=2.31e-11, lr=5e-5, val_loss=nan] 
Finish Validation
Epoch:1/100
Total Loss: nan || Val Loss: nan
Save best model to best_epoch_weights.pth
为何total loss和val loss第一轮就是nan了呢？
